### PR TITLE
Add Kaptain to DKP 2.2 release notes and enable Kaptain Docs

### PR DESCRIPTION
**Pull request by juliangieseke**
⚠️ this should be the last PR enabling Kaptain Docs for production.

## Jira Tickets

https://jira.d2iq.com/browse/D2IQ-87247

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4375.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ For more information, see the [air-gapped bundle](../../install/air-gapped) docu
 
 The unified DKP user interface provides a smooth experience independent of where you start your journey. Both DKP Essential and DKP Enterprise customers will have similar experiences in the User Interface, with DKP Enterprise customers gaining access to additional features and benefits simply by entering their DKP Enterprise [license key](../../licensing/).
 
+### Kaptain AI/ML, D2iQ’s AI/ML offering
+
+For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS, AKS, and Microsoft Azure EKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```
Comment (MessKon) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:63]:
```suggestion
For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS EKS, and Microsoft Azure AKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```

Comment (juliangieseke) [reply pages/dkp/kommander/2.2/release-notes/2.2.0/index.md]: lol good catch! I could say I just copied that text from somewhere else, but that'd just be me defending myself 🙈 😂 

https://github.com/mesosphere/dcos-docs-site/pull/4206/commits/5dd71aa8b96068619fd9335ab93addac4a0afc3d

File: config.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "DO_NOT_BUILD": [
       "dkp/kommander/2.3/**",
       "dkp/konvoy/2.3/**",
-      "dkp/kaptain/2.0.0/**"
+      "dkp/kaptain/2.1/**"
```
Comment (juliangieseke) [inline config.json:19]: note to self: this will end up conflicting with #4376

File: index.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ MS.metadata({
   dispatchDocsLatest: "1.4",
   kommanderDocsLatest: "2.2",
   konvoyDocsLatest: "2.2",
-  kaptainDocsLatest: "1.3.0",
+  kaptainDocsLatest: "2.2.0",
```
Comment (juliangieseke) [inline index.js:55]: same as above

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md [hunk 4]
```diff
@@ -58,6 +58,10 @@ For more information, see the [air-gapped bundle](../../install/air-gapped) docu
 
 The unified DKP user interface provides a smooth experience independent of where you start your journey. Both DKP Essential and DKP Enterprise customers will have similar experiences in the User Interface, with DKP Enterprise customers gaining access to additional features and benefits simply by entering their DKP Enterprise [license key](../../licensing/).
 
+### Kaptain AI/ML, D2iQ’s AI/ML offering
+
+For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS EKS and Microsoft Azure AKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:63]:
```suggestion
For better integration with DKP 2.2, users can launch Kaptain as a catalog application. It also supports other platforms such as Amazon AWS EKS and Microsoft Azure AKS. Kaptain extends D2iQ’s ability to support Kubernetes platforms beyond DKP. It further enables an organization to develop, deploy and run entire ML workloads in production, at scale, with consistency and reliability.
```

**Review by mmcelroy91 — APPROVED** [review]

File: config.json (hunk 2 above)
Comment (cmastr) [reply config.json:19]: I think there's another place we need to change @juliangieseke

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md (hunk 4 above)
Comment (cmastr) [reply pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:63]: @mmcelroy91 I think 'you' is better than 'users'.

File: config.json (hunk 2 above)
Comment (cmastr) [reply config.json:19]: https://github.com/mesosphere/dcos-docs-site/blob/jg/add-kaptain-to-dkp-2-rn/s3config.json#L13

Comment (juliangieseke) [reply config.json:19]: good catch, thanks! updated 👍

**Comment by juliangieseke** [comment]
@Colin1841 done 👍

File: pages/dkp/konvoy/2.2/release-notes/index.md [hunk 5]
```diff
@@ -58,6 +58,10 @@ For more information, see the [air-gapped bundle](../choose-infrastructure) docu
 
 The unified DKP user interface provides a smooth experience independent of where you start your journey. Both DKP Essential and DKP Enterprise customers will have similar experiences in the User Interface, with DKP Enterprise customers gaining access to additional features and benefits simply by entering their DKP Enterprise [license key](/dkp/kommander/2.2/licensing/).
 
+### Kaptain AI/ML, D2iQ’s AI/ML offering
+
+For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS EKS and Microsoft Azure AKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```
Comment (juliangieseke) [inline pages/dkp/konvoy/2.2/release-notes/index.md:63]:
```suggestion
For better integration with DKP 2.2, you can launch Kaptain as a catalog application. It also supports other platforms such as Amazon AWS EKS and Microsoft Azure AKS. Kaptain extends D2iQ’s ability to support Kubernetes platforms beyond DKP. It further enables an organization to develop, deploy and run entire ML workloads in production, at scale, with consistency and reliability.
```

**Comment by juliangieseke** [comment]
rebased onto latest main, lets see..

File: index.js (hunk 3 above)
Comment (juliangieseke) [inline index.js:55]:
```suggestion
  kaptainDocsLatest: "2.0.0",
```
was this 2.2 all the time? lol..

**Review by cmastr — APPROVED** [review]

**Pull request merged**